tests/staticfiles_tests/settings.py

```diff
@@ -24,5 +24,5 @@
     ],
     # In particular, AuthenticationMiddleware can't be used because
     # contrib.auth isn't in INSTALLED_APPS.
-    'MIDDLEWARE': [],
+    'MIDDLEWARE': ['django.middleware.common.CommonMiddleware'],
```

Comment:
Maybe it's better to put this override on the test method so it's easier to see the purpose?